crates/matrix-sdk-common/src/tracing_timer.rs

```diff
@@ -45,6 +43,8 @@ impl Drop for TracingTimer {
             return;
         }
 
+        let message = format!("_{}_ finished in {:?}", self.id, self.start.elapsed());
```

Comment:
Hmm, this means the elapsed time now inclues the time to check the tracing level is enabled. Could we call `.elapsed()` *before*, store it in a temporary local, and use it here, instead? This means we'd always compute `.elapsed()`, which would result in being more precise, at the cost of doing the system call to compute the "now" time, so I'm a bit torn myself with doing that…